uber/site_sections/summary.py

```diff
@@ -103,17 +103,40 @@ def staffing_overview(self, session):
         }
 
     @csv_file
-    def personalized_badges(self, out, session):
-        for a in session.query(Attendee).filter(Attendee.badge_num != 0).order_by('badge_num').all():
-            out.writerow([a.badge_num, a.badge_type_label, a.badge_printed_name or a.full_name])
-        for a in session.query(Attendee).filter(Attendee.badge_type == c.STAFF_BADGE,
-                                                Attendee.amount_extra >= c.SUPPORTER_LEVEL).order_by(Attendee.full_name).all():
-            out.writerow(['', 'Supporter', a.badge_printed_name or a.full_name])
+    def printed_badges_attendee(self, out, session):
+        uber.reports.printed_badge_report_type(badge_type=c.ATTENDEE_BADGE).run(out, session)
+
+    @csv_file
+    def printed_badges_guest(self, out, session):
+        uber.reports.printed_badge_report_type(badge_type=c.GUEST_BADGE).run(out, session)
+
+    @csv_file
+    def printed_badges_one_day(self, out, session):
+        uber.reports.printed_badge_report_type(badge_type=c.ONE_DAY_BADGE).run(out, session)
+
+    @csv_file
+    def printed_badges_staff(self, out, session):
+        uber.reports.personalized_badge_report_type()\
+            .run(out, session,
+                 sa.Attendee.badge_type == c.STAFF_BADGE,
+                 sa.Attendee.badge_num != 0,
+                 order_by='badge_num')
+
+    @csv_file
+    def printed_badges_supporters(self, out, session):
+        uber.reports.personalized_badge_report_type(include_badge_nums=False)\
+            .run(out, session,
+                 sa.Attendee.amount_extra >= c.SUPPORTER_LEVEL,
+                 order_by=sa.Attendee.full_name,
+                 badge_type_override='supporter')
 
     @multifile_zipfile
     def personalized_badges_zip(self, zip_file, session):
-        zip_file.writestr("test1.txt", "test1")
-        zip_file.writestr("test2.txt", "test2")
```

Comment:
Super-nitpicky style note: in general we use single quotes for strings in this codebase, and double quotes for strings which themselves contain single quotes.